### ISSUE TITLE
Is the plugin included in MonoDevelop these days?

### ISSUE THREAD

**Issue by xuhdev** [mikerochip/editorconfig-monodevelop#2]
https://github.com/editorconfig/editorconfig.github.com/issues/90 reports EditorConfig is supported by default now. Is it true? I'm very confused with the current status.

**Comment by mikerochip** [comment]
Yes it is true

TLDR: Apologies for the added confusion. I will archive this repo soon and add a note on why since this hasn't been maintained in a while and hasn't been needed since 2018.

Here's a list of why this is confusing:

* At the time I created this repo in Feb 2018 when there was no built-in support
* Built-in support was being worked on by the MonoDevelop maintainers while I was working on my plugin (I didn't realize this at the time)
* My plugin was released in Feb 2018
* Official support was released in May 2018
* Here is a permanent link to the release notes since the issue you referenced doesn't provide one: https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-mac-relnotes-v7.5#release-highlights
* My plugin provides an option to normalize EOLs or not on file open/save. This is a gap in the editorconfig spec. AFAICT no other editorconfig support (in any IDE) provides this option
* **My plugin is not in the MonoDevelop codebase in any way, shape, or form** it just happens to be the ONLY option for editorconfig support prior to the 7.5 release. However, this is all very old and outdated tooling at this point, so not many folks in the wild will probably care.

There's more context on MonoDevelop and Visual Studio for Mac and why it's so confusing. Partial history is here: https://en.wikipedia.org/wiki/MonoDevelop. However that wikipedia page doesn't do a good job of articulating the timeline or details of why this is such a confusing situation. Only Microsoft insiders and former Xamarin team members (the original creators and maintainers) know the whole story behind it, and that doesn't include me, unfortunately.

Here's some more context. Just want to re-iterate that this is **my POV as an outsider**:

* MonoDevelop was originally a cross-platform IDE intended to be built on top of the Mono runtime (an older cross-platform implementation of the then-Windows-specific .Net Framework). It actually has roots going back to 2006 and was the first rebrand to happen (from SharpDevelop -> MonoDevelop).
* Microsoft created a very confusing situation with Visual Studio for Mac. They effectively just rebranded the MonoDevelop IDE as Visual Studio for Mac, and made the installer specific to Mac. They didn't change core IDE functionality. As far as a user could tell, they just changed the way you install, and changed some file names and title bars. The VS for Mac IDE didn't do a great job hiding all of the references to MonoDevelop in it. The SharpDevelop -> MonoDevelop rebrand was much more comprehensive.
* This situation left the future of MonoDevelop on Windows and Linux VERY unclear. I'm just guessing here, but: I think because the userbase on those platforms was so small, they didn't bother creating clear messaging.
* The future of MonoDevelop and how it related to VS for Mac was also unclear at this point. MonoDevelop was still being maintained and as far as an outsider could tell from their github repo, and was still being used as the core of VS for Mac. The [MonoDevelop repo](https://github.com/mono/monodevelop) was archived in Oct 2021 but there's a note in the repo saying it hasn't been maintained since Jan 2020. So, that's when VS for Mac effectively became insourced instead of open source.
* I'm guessing Microsoft made an internal decision to rebuild VS for Mac? There's a bullet point on the MS help pages about using native macOS UI, which in my interpretation is a hint that they're rebuilding it from scratch and simply keeping the brand name. Here: https://docs.microsoft.com/en-us/visualstudio/productinfo/mac-roadmap#general

I'm sad that Microsoft isn't providing more clear context and messaging about this situation. Maybe internal politics were at play, maybe I just simply missed a blog post about it. I just wish they would provide a more permanent spot on their website articulating the history here since there are kind of just vague references to it on their official VS for Mac pages along with some outdated mono and MonoDevelop pages, which hint at it.
